libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/component/infopanel/InfoPanelHeaderComponent.kt

```diff
@@ -57,51 +49,15 @@ internal class InfoPanelHeaderComponent(
         }
 
         binding.routePreview.setOnClickListener {
-            coroutineScope.launch {
-                if (fetchRouteIfNeeded()) {
-                    store.dispatch(NavigationStateAction.Update(NavigationState.RoutePreview))
-                }
-            }
+            store.dispatch(Command.ShowRoutePreview)
         }
 
         binding.startNavigation.setOnClickListener {
-            coroutineScope.launch {
-                if (fetchRouteIfNeeded()) {
-                    store.dispatch(NavigationStateAction.Update(NavigationState.ActiveNavigation))
-                }
-            }
+            store.dispatch(Command.StartActiveNavigation)
```

Comment:
Is there a way to get feedback on potential errors when this action fails?

Reply:
Nope. That's [out of scope](https://github.com/mapbox/navigation-sdks/issues/1645).

Reply:
👌  yeah sounds good to me